src/encoding/schema/map.ts

```diff
@@ -424,3 +449,136 @@ export class StringMapSchema extends Schema {
     return map;
   }
 }
+
+/**
+ * Converts any RawBinaryString values to regular strings in a MsgpackEncodingData object.
+ *
+ * Note this conversion may be lossy if the binary data is not valid UTF-8.
```

Comment:
Should there be an parameter option to error rather than silently be lossy?

Reply:
Yes, after more consideration I think that's the safest thing to do. Working on that now

Reply:
Implemented in 561aa4db